src/main/java/seedu/address/logic/parser/PriorityCommandParser.java

```diff
@@ -34,11 +34,18 @@ public PriorityCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
                     PriorityCommand.MESSAGE_USAGE));
         }
-
+        if (argMultimap.getAllValues(PREFIX_NRIC).size() > 1) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    PriorityCommand.MULTIPLE_PARAMETER));
+        }
         if (!argMultimap.getValue(PREFIX_PRIORITY).isPresent()) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
                     PriorityCommand.MESSAGE_USAGE));
         }
+        if (argMultimap.getAllValues(PREFIX_PRIORITY).size() > 1) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    PriorityCommand.MULTIPLE_PARAMETER));
+        }
```

Comment:
I believe you can utilise the `verifyNoDuplicatePrefixes` command in `argumentMultiMap` like in `addCommandParser` 


![image](https://github.com/user-attachments/assets/b28bacb6-f769-4a7a-a3c3-77032a31e364)